code/modules/jobs/job_types/chaplain.dm

```diff
@@ -98,10 +93,6 @@
 			B.desc = "Happy, Full, Clean. Live it and give it."
 		if("subgenius")
 			B.name = "Book of the SubGenius"
-		if("toolboxia","greytide")
-			B.name = pick("Toolbox Manifesto","iGlove Assistants")
```

Comment:
I liked this one :(

Reply:
I could just remove homosexuality and the brain damage one, the general principle was that things that'd get you banned shouldn't be in this code, but admittedly the last two aren't very significant.

Reply:
yea

Reply:
greytide

Reply:
Went with this